src/components/dropdown/CustomRender.tsx

```diff
@@ -252,8 +268,12 @@ export const DropdownList = (props: any) => {
     }
     focusThis = focusThis || searchHeaderOption;
     // Clear the value if header option is selected
-    if (select?.state?.selectValue[0]?.searchHeader) {
-      select?.clearValue();
+    // TODO -> Add termSearch to async mode
+    if (select?.state?.selectValue) {
+      if(select?.state?.selectValue[0]?.searchHeader) {
```

Comment:
there is a risk of break if select.state.selectValue is empty

Reply:
done